gnmawsgr/tasks.py

```diff
@@ -129,11 +129,18 @@ def valid_fields(self):
 def download_callback(rq, current_progress,total):
     if rq.status != 'DOWNLOADING':
         rq.status = 'DOWNLOADING'
+
+    try:
+        percent = 100*float(current_progress)/float(total)
+
+    except ZeroDivisionError as e:
```

Comment:
you know what, i think we should actually run a differnet logger.info line here.  Something like, logger.warning("{itemid} Trying to download but last chunk size was zero".format(rq.item_id))